assets/scss/styles.scss

```diff
@@ -126,69 +152,52 @@ h1 {
 }
 
 /* Stats page */
-#stats {
+.stat_data {
+  h2 {
+    @include bold-24;
+  }
   table {
+    margin: 0 50px 50px;
     border-collapse: collapse;
     border-spacing: 0;
-    width: 100%;
-    margin-bottom: 50px;
+    box-sizing: border-box;
     thead {
       tr {
         th,
         td {
-          padding-top: 0;
-          padding-bottom: 16px;
+          padding: 0 1.25em 16px 0;
           white-space: nowrap;
           vertical-align: bottom;
-        }
-        th:last-child {
-          text-align: right;
+          &.numeric {
+            @include bold-19($tabular-numbers: true);
+            text-align: right;
+          }
+          &.centre_updated {
+            @include core-16;
+            color: red;
+            text-align: right;
+          }
+          &:last-child {
+            width: 10%;
+            padding-right: 0;
+          }
         }
       }
     }
-    thead,
     tbody {
+      background-color: $background-color;
       tr {
-        th {
-          &.numeric {
-            text-align: left;
-          }
-        }
-        td {
-          &.numeric {
-            font-family: "ntatabularnumbers", "nta", Arial, sans-serif;
-            font-size: 14px;
-            line-height: 1.1428571429;
-            font-weight: 400;
-            text-transform: none;
-            @media (min-width: 641px) {
-              font-size: 16px;
-              line-height: 1.25;
-            }
-          }
-        }
         th,
         td {
           @include core-19;
           padding: 0.75em 1.25em 0.5625em 0;
           text-align: left;
           color: #0b0c0c;
```

Comment:
is there nothing in the gov stylesheet for this colour? if not - should we even use it?